cf-agent/verify_packages.c

```diff
@@ -195,7 +195,7 @@ PromiseResult VerifyPackagesPromise(EvalContext *ctx, const Promise *pp)
             break;
 
         case PACKAGE_PROMISE_TYPE_OLD:
-            Log(LOG_LEVEL_VERBOSE,
+            Log(LOG_LEVEL_INFO,
                 "Using old package promise. Please note that this old "
```

Comment:
While here, perhaps we should update the messages a bit. "old" is not very specific, probably better to say `package_method` implementation or something